src/screens/CourseScreen/index.tsx

```diff
@@ -110,6 +126,13 @@ export const CourseScreen = () => {
 			>
 				<ChatBubbleIcon sx={{ color: '#e9ecef' }} />
 			</Button>
+			<FeedbackModal
+				opneModal={openModal}
+				handleOpenModal={handleOpenModal}

```

Comment:
no need to pass the handleOpenModal function to the modal, the state that determine whether the modal is ipen or not is enough.